netman/adapters/switches/cisco.py

```diff
@@ -291,6 +291,18 @@ def remove_dhcp_relay_server(self, vlan_number, ip_address):
         with self.config(), self.interface_vlan(vlan_number):
             self.ssh.do("no ip helper-address {}".format(ip_address))
 
+    def set_bond_trunk_mode(self, number):
+        try:
+            return self.set_trunk_mode(bond_name(number))
+        except UnknownInterface:
+            raise UnknownBond(number)
```

Comment:
This try catch is everywhere, in all bond calls in all adapters i presume... maybe it would be worth using a annotation?  something like

@unkown_interface_to_unknown_bond (award winning naming skills here...)